src/lib/make-toolbox-xml.js

```diff
@@ -310,13 +310,17 @@ const sound = function () {
     `;
 };
 
-const events = function () {
+const events = function (isStage) {
```

Comment:
Note that isStage is already being passed into this function at the bottom of this file.